src/main/java/de/plushnikov/intellij/plugin/processor/clazz/log/AbstractLogProcessor.java

```diff
@@ -24,18 +25,29 @@
  * @author Plushnikov Michail
  */
 public abstract class AbstractLogProcessor extends AbstractClassProcessor {
-
-  private final String loggerType;
-  private final String loggerInitializer;
-  private final String loggerCategory;
+  enum LoggerInitializerParameter {
+    TYPE, NAME, TOPIC, NULL;
+
+    @Nullable
+    static LoggerInitializerParameter find(@NotNull String parameter) {
+      switch(parameter) {
+        case "TYPE":
+          return TYPE;
+        case "NAME":
+          return NAME;
+        case "TOPIC":
+          return TOPIC;
+        case "NULL":
+          return NULL;
+        default:
+          return null;
```

Comment:
I think using "null" is a bad idea. Maybe you can add some default/unknown state to the enum instead?

Reply:
I will introduce UNKNOWN.